packages/paste-core/components/heading/src/index.tsx

```diff
@@ -13,7 +13,7 @@ export type HeadingStyle =
 
 interface Heading {
   as: asTags;
-  id?: never;
+  id?: string;
```

Comment:
I'm not sure why we wouldn't allow `id`. I need it for the site headings

Reply:
Oh right. That makes sense.